services/github/github-release.service.js

```diff
@@ -39,22 +92,107 @@ module.exports = class GithubRelease extends GithubAuthV3Service {
     }
   }
 
-  static render({ version, isPrerelease }) {
-    return {
-      message: addv(version),
-      color: isPrerelease ? 'orange' : 'blue',
+  static render({ version, sort, isPrerelease }) {
+    let color = 'blue'
+    color = sort === 'semver' ? versionColor(version) : color
+    color = isPrerelease ? 'orange' : color
+    return { label: 'release', message: addv(version), color }
```

Comment:
We don't need to set the label to `release` again here since it's already set as the default, or do you think there's value in having it explicitly inline here with the message and color?